README.md

```diff
@@ -4,18 +4,28 @@ Diglett is a command line tool that helps you detect packages with multiple vers
 
 ## Usage
 
-```
-npx diglett <optional path to project>
-```
-
 By default _diglett_ will ignore packages not imported by dependencies defined in the `dependencies` field of your `package.json`. If duplicate resolved packages found it will exit with code `1` and output the resolved versions:
 
 ```
 Found 1 duplicate dependency
 @material/animation with versions 3.1.0, 4.0.0.
 ```
 
-## Options
+### Yarn projects
+
+```
+npx diglett yarn <optional path to project>
+```
+
+### Yarn workspaces
+
+```
+npx diglett workspace <optional path to project> [--package <package name>]
```

Comment:
@oblador Considering the regular one is `diglett yarn`, wouldn't it make sense for this one to be `diglett yarn-workspace`?

Reply:
Fixed in https://github.com/oblador/diglett/pull/1/commits/4270bbaa5f65921ddef5b293f4693756282bec2a